druid-shell/src/hotkey.rs

```diff
@@ -260,14 +265,30 @@ impl From<SysMods> for RawMods {
     }
 }
 
-impl From<KeyCode> for KeyCompare {
-    fn from(src: KeyCode) -> KeyCompare {
-        KeyCompare::Code(src)
+impl IntoKey for Key {
+    fn into_key(self) -> Key {
+        self
+    }
+}
+
+impl<'a> IntoKey for &'a str {
```

Comment:
```suggestion
impl IntoKey for &str {
```

I believe you can now omit this lifetime.

Reply:
Yes. I wonder why this isn't a clippy lint. Maybe would be too noisy on existing codebases.